freqtrade/exchange/exchange.py

```diff
@@ -228,6 +228,16 @@ def precisionMode(self) -> str:
         """exchange ccxt precisionMode"""
         return self._api.precisionMode
 
+    def throw_exception_if_method_not_on_exchange(self, method_name: str):
+        """
+            Throws exception if method is implemented by the current exchange on ccxt
+            :param method_name: The method that may/may not exist for this exchange class
+        """
+        # api_method = hasattr(self._api, method_name, None)
+        if not self._api.describe()['has'][method_name]:
+            raise OperationalException(
+                f"{method_name}() has not been implemented on ccxt.{self.name}")
+
```

Comment:
This already exists as `exchange_has()`, which however does not raise, but returns False.

There's 2 applications for that - either check at startup to verify if a configuration is valid (check if certain special options exist - like `createMarketOrder`) - or to check during runtime and fall back to alternative means. 
Neither of the 2 benefit from having that method "raise" a "failing" exception (on top of the fact that you're risking keyerrors with the approach you're taking).